pkg/apis/openebs.io/v1alpha1/cas_volume.go

```diff
@@ -83,13 +83,13 @@ const (
 	CASConfigCVK CASVolumeKey = "cas.openebs.io/config"
 
 	// NamespaceCVK is the key to fetch volume's namespace
-	NamespaceCVK CASVolumeKey = "k8s.io/namespace"
+	NamespaceCVK CASVolumeKey = "openebs.io/namespace"
```

Comment:
Again, we need to see about cas cstor key.
So these should go in cas_cstor.go or written here?
If written here we may need to update the file name IMO.

Reply:
cas_cstor.go if its cstor & cas_pool.go if its pool based

Reply:
Sure. This answers my question.